src/js/actions/transform.js

```diff
@@ -1112,19 +1135,9 @@ define(function (require, exports) {
                 // existing layers.
                 return this.flux.actions.layers.addLayers(currentDoc, toIDs, true, false);
             } else {
-                var textLayers = currentDoc.layers.allSelected.filter(function (layer) {
-                        // Reset these layers completely because their impliedFontSize may have changed
-                        return layer.kind === layer.layerKinds.TEXT;
-                    }),
-                    otherLayers = currentDoc.layers.allSelected.filterNot(function (layer) {
-                        return layer.kind === layer.layerKinds.TEXT;
-                    }),
-                    textLayersPromise = this.flux.actions.layers.resetLayers(currentDoc, textLayers),
-                    otherLayersPromise = this.flux.actions.layers.resetBounds(currentDoc, otherLayers);
-
-                return Promise.join(textLayersPromise, otherLayersPromise);
+                return debouncedMoveHandler();
             }
-        }, this);
+        }.bind(this);
 
         _moveToArtboardHandler = synchronization.debounce(function () {
```

Comment:
We seemed to agree yesterday that `debounce` with 0 timeout was weird at best. Does this seem OK to you?